lib/tasks/fixes/ap_5603.rake

```diff
@@ -32,7 +32,7 @@ namespace :fixes do
     puts "----------------------------------------------------------"
   end
 
-  desc "AP-55603: remove unused proceeding type SE099E"
+  desc "AP-5603: remove unused proceeding type SE099E"
```

Comment:
As this has been run on all environments, can it be deleted?

Reply:
Depends if we want to leave for posterity or not. Personally, I thinks it has value to it as an audit trail. Also devs who may have an old seeded version locally may want/need to run it.